Gaijinizer patch/CommonEvents_102.txt

```diff
@@ -22,8 +24,10 @@ Hmmm, I dunno...#Ron
 
 普段友達がお世話になってるみたいだし、特別だよ？
 > CONTEXT: CommonEvents_102/27/Dialogue/enemy
-Well, It looks like your friends are usually
-indebted, is it special?#MTLed
+Well...♡
+
+It looks like my friends are in your debt, so
+I'll make an exception this time okay?#StrateUnsure
```

Comment:
Lampas, the dialogues seem to, at least from what I've seen, be organized from least affection to most affection. So it starts with outright refusal, to indecision, to asking for something in return, to a free pass, to giving you money. This would be the indecision step.